### DISCUSSION TITLE
Why use such a weird ID LSPosed.github.io for Shamiko

### DISCUSSION THREAD

**Discussion by Originalimoc** [LSPosed/LSPosed#2058] [Ideas]
Hard to search using its function.
Just search "hide root magisk" on Google.
Search Shamiko in page, way down there.

**Comment by yujincheng08** [comment]
LSPosed.github.io is not for shamiko.

**Comment by Originalimoc** [comment]
What do you mean?
https://github.com/LSPosed/LSPosed.github.io/releases

**Comment by Originalimoc** [comment]
BTW what's the story behind "Momo can detect any module that requires not to be unloaded."?
